global/css/comments.css

```diff
@@ -1,401 +1,295 @@
-    font-family: Comfortaa;
-    box-sizing: border-box;
-    width: 100%;
-    word-break: break-word;
-}
 
-.comments__input-box__emoji {
-    border-radius: 0px !important;
-    background-color: rgba(var(--accentdark), 0.5) !important;
-    display: flex;
-    align-items: center;
-    justify-content: center;
+#comments__box, .comments__area {
+    max-width: 100%;
+    overflow: hidden;
 }
+.comments__comment-content {
+    position: relative;
+    z-index: 1;
+    border-radius: 10px;
+    padding: 16px;
+    align-self: stretch;
+    box-sizing: border-box;
+    background: linear-gradient(hsl(var(--accentdark_hue), var(--accentdark_saturation), calc(25% * var(--accentdark_valueoffset))), hsl(var(--accentdark_hue), var(--accentdark_saturation), calc(21% * var(--accentdark_valueoffset))));
 
-.comments__input-box__emoji:hover {
-    transform: scale(1);
 }
 
-.comments__input-box__text, .comments__input-box__send {
-    border: 0px;
-    outline: transparent;
-    background-color: #0000;
-    color: white;
-    min-height: 20px;
+.comments__area {
+    width: 100%;
 }
 
-.comments__post-box {
-    display: flex;
-    width: 100%;
+.comments_emoji_popup_conainer {
+    position: absolute;
+    z-index: 4;
+    margin-top: 4px;
 }
 
-.comments__input-box__send {
-    padding: 0px 13px;
-    background-color: rgba(var(--accentdark), 1);
-    font-size: 16px;
-    height: 100%;
-    margin: 0px;
-    border-radius: 0px 8px 8px 0px;
-    transition: all 0.1s ease;
+.comments_emoji_popup_conainer-hidden {
+    display: none;
 }
 
-.comments__area {
-    width: 100%;
-    display: flex;
-    flex-direction: column;
+.comments__input-box__emoji {
+    margin: 0px 12px;
 }
 
+.comments__post-disabled {
+    opacity: 0.5;
+    pointer-events: none;
 
-.comments__comment {
-    display: flex;
 }
 
-.comments__comment-div {
-    background: linear-gradient(#fff1, #0001), linear-gradient(rgba(var(--accentdark), 0.2), rgba(var(--accentdark), 0.3));
-    border-radius: 8px;
-    display: flex;
-    flex-direction: column;
-    width: 100%;
-    margin-left: 17px;
-    flex: 1;
-    max-width: 100%;
-    min-width: 0;
-    align-self: stretch;
-}
 
-.comments__comment-bottom {
-    background: linear-gradient(#0001, #fff1), linear-gradient(rgba(var(--accentdark), 0.6), rgba(var(--accentdark), 0.3));
-    border-radius: 0px 0px 8px 8px;
+#oReplyBox {
+    padding-left: 20px;
+    padding-top: 8px;
+    padding-bottom: 8px;
     display: flex;
 }
-.comments__comment-top-text {
-    font-family: var(--body-font);
-    opacity: 0.8;
-    font-weight: 400;
-    overflow: hidden;
-    font-size: 16px;
-    white-space: pre-line;
+.comments__comment-content img {
+    max-width: 10%;
 }
 
-.comments__comment-vote, .comments__comment-reply, .comments__comment-delete, .comments__comment-report {
-    width: 40px;
-    background: linear-gradient(#fff1, #0001), linear-gradient(rgba(var(--accentdark), 1), rgba(var(--accentdark), 1));
+.comments__comment-votes {

```

Comment:
Consider a hover and selected event for upvoting a comment.

Reply:
have done